test/integration.lean

```diff
@@ -35,3 +37,28 @@ example : ∫ x in 0..2, -exp x = 1 - exp 2 := by simp
 /-- linear combinations (e.g. polynomials) -/
 example : ∫ x : ℝ in 0..2, 6*x^5 + 3*x^4 + x^3 - 2*x^2 + x - 7 = 1048 / 15 := by norm_num
 example : ∫ x : ℝ in 0..1, exp x + 9 * x^8 + x^3 - x/2 + (1 + x^2)⁻¹ = exp 1 + π / 4 := by norm_num
+
+/-! ### Functions composed with multiplication by and/or addition of a constant -/
+
+/-- many examples are computable by `norm_num` -/
+example : ∫ x in 0..2, -exp (-x) = exp (-2) - 1 := by norm_num
```

Comment:
I see the rest of the file already does it, but strictly `--` makes this a docstring for just this first example. That doesn't matter for now, but will matter if we want to include this in doc-gen somehow (which I think would be a great idea)

Reply:
I see. I'll change this throughout the file.